pipelines/push-disk-images-to-marketplaces/push-disk-images-to-marketplaces.yaml

```diff
@@ -227,6 +227,8 @@ spec:
           value: "$(tasks.collect-data.results.data)"
         - name: cloudMarketplacesSecret
           value: "$(tasks.collect-marketplacesvm-secret.results.cloudMarketplacesSecret)"
+        - name: prePush
+          value: "$(tasks.collect-marketplacesvm-secret.results.prePush)"
```

Comment:
Would it make sense to rename `collect-marketplacesvm-secret` to `collect-marketplacesvm-params` since it's now collecting more than just the secret?

Reply:
Sure, I can do that!

Reply:
Renamed